openfisca_france/model/prelevements_obligatoires/prelevements_sociaux/cotisations_sociales/allegements.py

```diff
@@ -341,9 +343,20 @@ def compute_allegement_fillon(simulation, period):
     # Ce montant est majoré de 10 % pour les entreprises de travail temporaire
     # au titre des salariés temporaires pour lesquels elle est tenue à
     # l’obligation d’indemnisation compensatrice de congés payés.
-    Pf = simulation.legislation_at(period.start).cotsoc.exo_bas_sal.fillon
-    seuil = Pf.seuil
-    tx_max = (Pf.tx_max * not_(majoration) + Pf.tx_max2 * majoration)
+
+    fillon = simulation.legislation_at(period.start).prelevements_sociaux.fillon
+
+    # Du 2003-07-01 au 2005-06-30
+    if date(2003, 7, 1) <= period.start.date <= date(2005, 6, 30):
```

Comment:
It's actually the same: we import `datetime.date` in `base.py`. So we can directly use `date` everywhere.